docs/daemon/validator.mdx

```diff
@@ -0,0 +1,36 @@
+---
+sidebar_position: 4
+sidebar_label: "Validators"
+id: daemon-setup-validators
+title: Becoming a validator
+description: How to become a validator node
+slug: /daemon/validator
+---
+
+The Kwil network uses CometBFT, the underlying consensus engine of the Cosmos SDK, for Practical Byzantine Fault Tolerant (PBFT) consensus.
+Validators in CometBFT are responsible for producing new blocks in the blockchain. They do so by participating in the consensus proposal by broadcasting votes containing cryptographic signatures produced by signing the messages using the validator's private key. 
+
+There are 2 ways to become a validator:
+1. Pre-establish them in the genesis file before starting the Kwil network.
+2. Going through a consensus-based join process.
+
+## Genesis File
+The Kwil network can be deployed with a pre-determined set of validators by including them in the genesis file. These genesis validators must be specified under the section `"validators"` in the genesis file. For more information, refer to the [genesis validators guide](./config/genesis#genesis-validators)
+
+## Federated Consensus-Based Join 
+
+If a node wants to become a validator post-network deployment, it can do so by undergoing a PBFT consensus-based approval process. The process of joining the network as a validator through federated consensus involves the following steps:
+
+1. A node that wishes to become a validator must issue a join request transaction signed by its private key. Join requests can be issued using the `kwil-admin` command line tool. For more information, refer to the [join request guide](../../admin/validators.mdx#join).
+
+2. Upon receiving the join request, existing validators must determine if the node should be granted validator status.  Validator nodes that decide to approve must issue an approval request for the joiner, signed by the validator's private key. This approval request is broadcasted as a blockchain transaction. Refer to the [approval request guide](../../admin/validators.mdx#approve) for more information on issuing approval requests using `kwil-admin` tool.
+
+3. The join requests remain active for the duration of `join_expiry` defined in the genesis file. The join request will expire if 2/3rd of the current validators don't approve the join request within this duration. The status of the Join request can be queried using the `kwil-admin` command line tools. Refer to the [join request status guide](../../admin/validators.mdx#show-join-request-status) for more information.
```

Comment:
"Join" should not be capitalized.  Also, it is the `kwil-admin` command line tool, not "tools"